build.gradle.kts

```diff
@@ -6,7 +6,7 @@ plugins {
 	id("maven-publish")
 	id("signing")
 	id("net.kyori.blossom") version "1.3.1"
-	id("io.github.gradle-nexus.publish-plugin") version "1.3.0"
+	id("io.github.gradle-nexus.publish-plugin") version "2.0.0-rc-1"
```

Comment:
Please revert this change. Modifying the build file will require a review from seeseemelk before being merged. We also shouldn't switch to pre-release versions before they're ready.